Makefile

```diff
@@ -83,7 +83,7 @@ CORE_FILES=const.js config.js io.js main.js lib.js ide.js pci.js floppy.js \
 	   cpu.js debug.js \
 	   elf.js kernel.js
 LIB_FILES=9p.js filesystem.js jor1k.js marshall.js utf8.js
-BROWSER_FILES=screen.js keyboard.js mouse.js speaker.js serial.js \
```

Comment:
What's the issue with speaker.js?

Reply:
Looks like version problem of Closure compiller, or Nodejs, or Reflect:

```
src/browser/speaker.js:487: ERROR - [JSC_UNDEFINED_VARIABLE] variable AudioWorkletProcessor is undeclared
            var self = Reflect.construct(AudioWorkletProcessor, [], DACProcessor);
                                         ^^^^^^^^^^^^^^^^^^^^^

src/browser/speaker.js:766: ERROR - [JSC_UNDEFINED_VARIABLE] variable AudioWorkletNode is undeclared
        this.node_processor = new AudioWorkletNode(this.audio_context, "dac-processor",
                                  ^^^^^^^^^^^^^^^^

2 error(s), 0 warning(s)
Makefile:100: recipe for target 'build/v86_all.js' failed
```

Reply:
It is not priority.